tests/common/plugins/conditional_mark/tests_mark_conditions_drop_packets.yaml

```diff
@@ -143,5 +143,12 @@ drop_packets/test_drop_counters.py::test_src_ip_link_local:
     conditions:
       - "asic_type=='cisco-8000'"
 
+drop_packets/test_drop_counters.py::test_non_routable_igmp_pkts[vlan_members:
```

Comment:
Please keep the test case in alphabetic order.

Reply:
Done. Thanks!